expyfun/_input_controllers.py

```diff
@@ -84,6 +99,17 @@ def _on_pyglet_keypress(self, symbol, modifiers, emulated=False):
             this_key = this_key.lstrip('_').lstrip('NUM_')
         self._keyboard_buffer.append((this_key, key_time))
 
+    def _on_pyglet_keyrelease(self, symbol, modifiers, emulated=False):
```

Comment:
@drammock  The only difference is that they use `self._keyboard_buffer_releases`, which contains only the release events.